.github/workflows/release-winget.yaml

```diff
@@ -0,0 +1,38 @@
+name: "release-winget"
+on:
+  release:
+    types: [released]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - id: update-winget
+      name: Update winget repository
+      uses: mjcheetham/update-winget@v1.1
+      with:
+        id: Microsoft.Git
+        token: ${{ secrets.WINGET_TOKEN }}   
+        releaseAsset: Git-\[0-9.vfs]*\-64-bit.exe
+        manifestText: |
+          PackageIdentifier: {{id}}
+          PackageVersion: {{version}}
+          PackageName: Microsoft Git
+          Publisher: Microsoft Corporation
+          Moniker: microsoft-git
+          PackageUrl: https://aka.ms/ms-git
+          Tags: [ microsoft/git ]
+          License: Copyright (C) Microsoft Corporation
+          ShortDescription: |
+            Git distribution to support monorepo scenarios.
+            Note - This is not Git for Windows. Unless you are working in a monorepo and require
+            specific Git modifications, please run `winget install git` to start using Git for Windows.
```

Comment:
I want to call attention to this bit, as it will be the forward-facing description to users browsing the `winget` listing.

I think my only comment is to replace `Note -` with `Note:`.

Reply:
I originally had it as `Note:` but ran into some issues with `winget validate`. I'm sure there's an easy way to escape though, I'll experiment a bit.

Reply:
Nvm, I lied, it seems fine when we have a multi-line description like this.